src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -219,15 +337,120 @@ private function isDeclareStatement(Tokens $tokens, $index)
         return false;
     }
 
-    private function runHelperFixers(\SplFileInfo $file, Tokens $tokens)
+    private function resolveConfig(array $configuration)
     {
-        /** @var AbstractAlignFixerHelper $helper */
-        foreach ($this->alignFixerHelpers as $helper) {
-            if ($tokens->isChanged()) {
-                $tokens->clearEmptyTokens();
+        $this->configuration = [];
```

Comment:
`resolveConfig` is called in `configure`, which already assing value to `$this->configuration`. why we totally override it here? I don't get it.

Reply:
I couldn't fit in the BC layer for the old configuration in the current configuration resolving logic, if anyone knows a better way please send a PR to my branch.

Reply:
>in such case, does it make sense to still call parent::configure ?

I think so; as the parent does check of the configuration that the fixer itself doesn't.
Utest should be close to all paths, but a test with if `<=>` configured by default on PHP > 7 is lacking for example, there might be more missing tests.